drafts/draft-ietf-ippm-ioam-data.xml

```diff
@@ -1896,10 +1894,11 @@ be 4-octet aligned:
             <t>IOAM E2E-Type</t>
 
             <t>IOAM Namespace-ID</t>
-          </list>will contain the current set of possibilities defined in this
-        document. New registries in this name space are created via RFC
-        Required process as per <xref target="RFC8126"/>.</t>
+          </list></t>
 
+        <t>New registries in this group can be created via RFC
+        Required process as per <xref target="RFC8126"/>.</t>
+		
         <t>The subsequent sub-sections detail the registries herein
         contained.</t>
       </section>
```

Comment:
Since you got rid of "<section ... >" above, this also needs to be deleted. Each of the sub-sections below belong inside the IANA Considerations section.